force-app/main/default/lwc/libsD3/__tests__/libsD3.test.js

```diff
@@ -0,0 +1,44 @@
+import { createElement } from 'lwc';
+import LibsD3 from 'c/libsD3';
+import { loadScript, loadStyle } from 'lightning/platformResourceLoader';
+
+describe('c-libs-d3', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Clear mocks so that every test run has a clean implementation
+        jest.clearAllMocks();
+    });
+
+    it('contains a svg element for D3', () => {
+        // Create initial element
+        const element = createElement('c-libs-d3', {
+            is: LibsD3
+        });
+        document.body.appendChild(element);
```

Comment:
@trevor-bliss @kevinv11n How should we test this? Adding `lwc:dom` to a tag throws `MutationObserver should not be mocked outside of the jest test environment`.

Reply:
Does the ChartJs test work? That also has the `lwc:dom` tag.